spec/connection_management_spec.rb

```diff
@@ -57,10 +53,10 @@ def with_connection(db)
   end
 
   it "inherits prepared_statements" do
-    ActiveRecord::Base.configurations[Rails.env]["prepared_statements"] = false
```

Comment:
We can’t do that anymore with Rails 6.1+ (it’s a frozen hash) so the simplest thing to do was to load another config file with prepared statements disabled.

Reply:
sounds good to me!